### PR TITLE
{bio} [foss,2015b] HTSeq-0.6.1p1

### PR DESCRIPTION
**Pull request by verdurin**
Port HTSeq and matplotlib dep to foss-2015b.

Changed moduleclass of HTSeq from `data` to `bio`.

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: ok to test

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
@verdurin: missing deps:

```
Irresolvable dependencies encountered: freetype/2.6.2-foss-2015b, libpng/1.6.21-foss-2015b
```

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/18274673ada939d5a615 for a full test report.

File: easybuild/easyconfigs/l/libpng/libpng-1.6.21-foss-2015b.eb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+toolchainopts = {'pic': True}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [('zlib', '1.2.8')]
+
+configopts = "--with-pic"
```
Comment (boegel) [inline easybuild/easyconfigs/l/libpng/libpng-1.6.21-foss-2015b.eb]: other libpng 1.6.21 easyconfigs include a sanity check, please do so here too?

```python
majminver = ''.join(version.split('.')[:2])
sanity_check_paths = {
    'files': ['include/pngconf.h', 'include/png.h', 'include/pnglibconf.h', 'lib/libpng.a',
              'lib/libpng.%s' % SHLIB_EXT, 'lib/libpng%s.a' % majminver, 'lib/libpng%s.%s' % (majminver, SHLIB_EXT)],
    'dirs': ['bin', 'include/libpng%s' % majminver, 'share/man'],
}
```

Comment (verdurin) [inline easybuild/easyconfigs/l/libpng/libpng-1.6.21-foss-2015b.eb]: It's not my fault that you're stricter now than you used to be...

Comment (boegel) [inline easybuild/easyconfigs/l/libpng/libpng-1.6.21-foss-2015b.eb]: It's not so much about being strict, but keeping easyconfigs for the same software verison in sync...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/5a9da8e2895f6ca99216 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @verdurin!

**Pull request merged**